mocha.el

```diff
@@ -85,10 +96,12 @@ If MOCHA-FILE is specified run just that file otherwise run
 MOCHA-PROJECT-TEST-DIRECTORY.
 
 IF TEST is specified run mocha with a grep for just that test."
-  (let ((path (or mocha-file mocha-project-test-directory))
+  (let* ((path (or mocha-file mocha-project-test-directory))
         (target (if test (concat "--grep \"" test "\" ") ""))
         (node-command (concat mocha-which-node (if debug (concat " --debug=" mocha-debug-port) "")))
         (options (concat mocha-options (if debug " -t 21600000"))))
```

Comment:
Perhaps we should refactor this whole function and at the same time write some tests for it as it is such a crucial part? But that would be for another PR.